File: nattybear/level1/문자열-내-p와-y의-개수/solution.js
function solution(s) {
  let p = 0;
  let y = 0;
  for (let c of s) {
    c = c.toLowerCase();
    if (c == 'p')
      p++;
    else if (c == 'y')
      y++;
  }
  return p == y;
}

File: nattybear/level1/문자열-내-p와-y의-개수/Solution.java
class Solution {
  boolean solution(String s) {
    int p = 0;
    int y = 0;
    for (int i = 0; i < s.length(); i++) {
      char c = Character.toLowerCase(s.charAt(i));
      if (c == 'p')
        p++;
      else if (c == 'y')
        y++;
    }
    return p == y;
  }
}
